src/Runtime/Core/Collections/Option.cs

```diff
@@ -2,11 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Collections.Generic;
-
-namespace Microsoft.ML.Probabilistic.Utilities
+namespace Microsoft.ML.Probabilistic.Core.Collections
```

Comment:
Should be Microsoft.ML.Probabilistic.Collections

Reply:
Will fix it that.

I did a move using an IDE. I think it would be a good idea (in a separate PR) to rename `Core/Collections` folder to `Collections`. Generally, mirroring namespaces and directory structure is convenient.